### PR TITLE
CFE-3920: Cleaned up policy related to versions prior to 3.12

### PR DESCRIPTION
**Pull request by nickanderson**
Now that 3.12.x is not under standard support, we can remove anything that was
not used in 3.12. Things used in 3.12 need to remain to cover the case where
someone is upgrading to 3.15.x because 3.12.x was supported when 3.15.0 was
released.

This change redacts policy related to cf-consumer and cf-redis-server.

Ticket: CFE-3920
Changelog: Title

### CODE REVIEW

**Review by olehermanse — APPROVED** [review]

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7571/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7571/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7571/

File: lib/README.md [hunk 1]
```diff
@@ -3,9 +3,6 @@ referred to as the Common Open Promise Body Library.
 
 Layout of this `lib` directory:
 
-* `3.6/*.cf`: Modularized version of the CFEngine Standard Library for
```
Comment (craigcomstock) [inline lib/README.md:6]: how is this related to 3.12? Did we make a switch from 3.6/*.cf to lib/*.cf at that point?

Comment (craigcomstock) [reply lib/README.md:6]: or was 3.6 supported when 3.12 was released?

**Review by craigcomstock — COMMENTED** [review]
looks good! one question about 3.6/*.cf

File: lib/README.md (hunk 1 above)
Comment (nickanderson) [reply lib/README.md:6]: It's not related to 3.12 at all, it's just another cleanup that I found while I was looking around. This is back from before we had macros to guard against future syntax and because of that we split the stdlib into version specific trees so that there were no new functions loaded by old agents that didn't understand the macros to avoid parsing functions and language features they didn't know about. 

It seemed too small to have in a separate PR, that's why its in this PR in a separate commit and that commit doesn't have a Ticket keyword :)
